src/lib/common/defaultValues.h

```diff
@@ -28,8 +28,9 @@
 
 
 
-#define  DEFAULT_SERVICE_PATH_UPDATES  "/"
-#define  DEFAULT_SERVICE_PATH_QUERIES  "/#"
-
+#define  DEFAULT_SERVICE_PATH_UPDATES         "/"
+#define  DEFAULT_SERVICE_PATH_QUERIES         "/#"
+#define  DEFAULT_SERVICE_KEY_FOR_METRICS      "default-service"
```

Comment:
Probably metrics defaults should be at MetricsManager.h (or even MetricsManager.cpp?) as per encapsulation/modularity it doesn't make too much sense to be used outise there.